resource.go

```diff
@@ -108,6 +116,22 @@ func Merge(fs ...Resource) (Resource, error) {
 			return nil, fmt.Errorf("gid does not match: %v != %v", f.GID(), prototype.GID())
 		}
 
+		if f.ATime() != prototype.ATime() {
+			return nil, fmt.Errorf("atime does not match: %v != %v", f.ATime(), prototype.ATime())
```

Comment:
I'm a little worried this will be racy. If a hardlinked file is accessed in between the `Stat`s, atime wouldn't match. The existing checks for things like mode and UID/GID are much less likely to change in the narrow window between two `Stat`s.

Should we avoid considering atime when merging hardlinks?